client/db/src/bench.rs

```diff
@@ -525,6 +505,40 @@ impl<B: BlockT> StateBackend<HashFor<B>> for BenchmarkingState<B> {
 		*self.whitelist.borrow_mut() = new;
 	}
 
+	fn get_read_and_written_keys(&self) -> Vec<(Vec<u8>, u32, u32, bool)> {
```

Comment:
Docs plz? (especially on the return type)

Reply:
```suggestion
	/// Returns `(key, reads, writes, whitelisted)`. 
	fn get_read_and_written_keys(&self) -> Vec<(Vec<u8>, u32, u32, bool)> {
```